ple.py

```diff
@@ -39,11 +31,20 @@
 from WelcomeView import WelcomeView
 from CreditsView import CreditsView
 
+import gi
+from gi.repository import Gtk, Pango
+gi.require_version('Gtk', '3.0')
+
+installed_dir = os.path.abspath(os.path.dirname(__file__))
+os.chdir(installed_dir)
```

Comment:
Are you confident that none of the imports above used the current working directory?  That would be a tough thing to prove.